### PR TITLE
Fixes found during acceptance tests

### PR DESCRIPTION
**Pull request by shlomizadok**
During acceptance tests, we have found some issues:
* as `attr_accessible` was introduced into foreman core, we needed to add some of our own :)
* UI improvements 
  * make arf_reports#index look like config_reports#index
  * In Policiy schedule, change days selection to days of month
  * Restore the parse_html path which was accidentally deleted from routes (policy -> show guide)
  * Fix sorting on arf_reports#index
* N+1 Queries (!!)

### CODE REVIEW

**Comment by shlomizadok** [comment]
[test]

**Comment by shlomizadok** [comment]
@xprazak2 - any chance to test this?

File: app/models/foreman_openscap/arf_report.rb [hunk 1]
```diff
@@ -5,6 +5,7 @@ class ArfReport < ::Report
     include Taxonomix
     include OpenscapProxyExtensions
 
+    # attr_accessible :host_id, :reported_at, :status, :metrics
```
Comment (xprazak2) [inline app/models/foreman_openscap/arf_report.rb]: When trying to upload a report: ActiveModel::MassAssignmentSecurity::Error: Can't mass-assign protected attributes: host, reported_at, status, metrics

We will need to add openscap_proxy in there ass well

Comment (shlomizadok) [inline app/models/foreman_openscap/arf_report.rb]: This was fixed with https://github.com/theforeman/foreman/pull/2961
fetch & rebase from foreman/develop :)

File: app/views/policies/_form.html.erb [hunk 2]
```diff
@@ -32,7 +32,7 @@
                    { :include_blank => _('Choose period') },
                    { :onchange => 'period_selected(this)' }) %>
       <%= select_f(f, :weekday, days_of_week_hash, :first, :last, :include_blank => _('Choose weekday')) %>
-      <%= text_f(f, :day_of_month, :help_inline => _('Number of a day in month, note that not all months have same count of days')) %>
+      <%= select_f(f, :day_of_month, (1..31).to_a, :to_i, :to_s, :help_inline => _('Number of a day in month, note that not all months have same count of days')) %>
```
Comment (xprazak2) [inline app/views/policies/_form.html.erb]: I cannot see the help text anywhere http://i.imgur.com/Oms30la.png

File: app/models/foreman_openscap/arf_report.rb (hunk 1 above)
Comment (xprazak2) [inline app/models/foreman_openscap/arf_report.rb]: Updating foreman/develop helped, could you just make a small change? s/openscap_proxy_id/openscap_proxy/ in here:
https://github.com/theforeman/foreman_openscap/blob/master/app/models/concerns/foreman_openscap/openscap_proxy_extensions.rb#L7

**Comment by xprazak2** [comment]
Looking good :+1: , just found one more thing - not sure if it is new or we just did not notice before: When on arf reports index page, try sorting on Host. I get `Can not eagerly load the polymorphic association :assetable`

**Comment by shlomizadok** [comment]
@xprazak2 - thanks for reviewing. fixed all your comments

**Comment by xprazak2** [comment]
Everything seems to be working now, no more comments

**Pull request merged**